metadata-db-app/src/migrations/054_catalog_item_acl_fix.clj

```diff
@@ -0,0 +1,67 @@
+(ns migrations.054-catalog-item-acl-fix
+  (:require [clojure.java.jdbc :as j]
+            [clojure.edn :as edn]
+            [cmr.common.util :as util]
+            [config.migrate-config :as config]
+            [config.mdb-migrate-helper :as h]
+            [cmr.metadata-db.data.oracle.providers :as providers]
+            [clojure.string :as string]))
+
+(defn- get-concept-ids
+  "For a given provider and entry-titles in collection-identifier, returns list of collection concept ids"
+  [provider-id collection-identifier]
+  (when-let [entry-titles (get collection-identifier :entry-titles)]
+    (let [provider (-> (h/get-provider provider-id)
+                       providers/dbresult->provider)
+          t (h/get-provider-collection-tablename provider)]
+      (for [entry-title entry-titles
+            :let [result (j/query (config/db) [(format "select distinct concept_id from metadata_db.%s where entry_title = ?" t)
+                                               entry-title])]]
+          (:concept_id (first result))))))
+
+(defn up
+  "Migrates the database up to version 54."
+  []
+  (println "migrations.054-add-coll-ids-catalog-acls up...")
+  (doseq [result (h/query "select * from cmr_acls where acl_identity like 'catalog-item%'")]
+    (println result)
```

Comment:
Didn't you have an issue with `h/query` and `println` in a prior migration for this work? Are you sure it is ok for this?